iib/workers/config.py

```diff
@@ -26,6 +26,11 @@ class Config(object):
     )
     iib_request_logs_level = 'DEBUG'
     iib_required_labels = {}
+    # Configuration for dogpile.cache
+    # To disable dogpile.cache you can use 'dogpile.cache.null' backend
+    iib_dogpile_backend = 'dogpile.cache.memcached'
```

Comment:
I would prefer that the default be to have caching disabled. This would make this an opt-in feature by the person deploying IIB.

Reply:
yes, this might be good approach, thanks.